### DISCUSSION TITLE
No Reusability with certain models

### DISCUSSION THREAD

**Discussion by SwissCheese15** [pmndrs/gltfjsx#192] [Q&A]
## Only One Model Showing

Hello Community

I have a project where I really need to be able to reuse the models that I load. But with my models (By Quaternius) it doesn't work.
I can't insert the Model multiple times. When I try to load the same Component again with different positions, only the last one shows up. I'm using the Component that **GLTFJSX** made for me. I have been at this for a long time now, pls help.

### Two Components:
<img width="396" alt="Bildschirmfoto 2023-03-15 um 20 34 07" src="https://user-images.githubusercontent.com/110164826/225423508-1beb497d-5860-4995-b3ca-f02586886946.png">

### Only one Cactoro
<img width="400" alt="Bildschirmfoto 2023-03-15 um 20 37 43" src="https://user-images.githubusercontent.com/110164826/225423984-c68deba8-606f-4b25-8179-261db4477ff5.png">

I can't upload the GLTF file here but this is the link where I got it. It doesn't work with any of them:
https://drive.google.com/drive/folders/1E_UWCNhXCUIC4P8_MnaHFT-_Yi-JOzC4

This is the project I'm working on:
https://github.com/SwissCheese15/RPG

Parent Component:
https://github.com/SwissCheese15/RPG/blob/main/src/Components/NPC/Npc.jsx

Model-Component:
https://github.com/SwissCheese15/RPG/blob/main/src/Components/NPC/Cactoro.jsx

**Comment by Shao-fract** [comment]
Hi same problem here.  Models are in fact  loading ( 6 from an array in my case) but all on the position of the last declared one.
Onclick method is working well as any other props. Must be some shared geometry somewhere.
If anyone have an idea.

**Comment by Shao-fract** [comment]
OK find out that solution have been found here : [((https://github.com/pmndrs/react-three-fiber/issues/245))]
